pkg/tsdb/grafana-postgresql-datasource/testdata/table/types_datetime.golden.jsonc

```diff
@@ -201,22 +201,10 @@
           ],
           null,
           null,
-          [
-            567000,
-            0
-          ],
-          [
-            567000,
-            567000
-          ],
-          [
-            432000,
-            0
-          ],
-          [
-            432000,
-            432000
-          ],
+          null,
+          null,
+          null,
+          null,
```

Comment:
Same question about these

Reply:
this related to the change changing the Time type to string which means no nanoseconds.